modules/claims_api/app/services/claims_api/bgs_to_lighthouse_claims_mapper_service.rb

```diff
@@ -0,0 +1,74 @@
+# frozen_string_literal: true
+
+module ClaimsApi
+  class BGSToLighthouseClaimsMapperService < ClaimsApi::Service
+    attr_accessor :bgs_claims, :lighthouse_claims
+
+    def initialize(bgs_claims:, lighthouse_claims:)
+      @bgs_claims        = bgs_claims
+      @lighthouse_claims = lighthouse_claims
+    end
+
+    def process
+      return [] unless claims_exist?
+
+      mapped_claims = map_claims
+      mapped_claims = append_remaining_lighthouse_claims(mapped_claims: mapped_claims) if lighthouse_claims.present?
+      mapped_claims
+    end
+
+    private
+
+    def claims_exist?
+      return true if bgs_claims.key?(:bnft_claim_detail) || lighthouse_claims.present?
+
+      false
+    end
```

Comment:
I'm not super hip to ruby conventions, but this could also be

```suggestion
    def claims_exist?
      bgs_claims.key?(:bnft_claim_detail) || lighthouse_claims.present?
    end
```

Reply:
I always forget cool stuff like that!  😂 
[Updated here.](https://github.com/department-of-veterans-affairs/vets-api/pull/7395/commits/906e5fea96187810333b059fce150e92813d4a31)